xml/System.Windows.Media.Effects/BitmapEffect.xml

```diff
@@ -164,7 +164,7 @@
       <Parameters />
       <Docs>
         <summary>Creates a handle to an [IMILBitmapEffect](http://msdn.microsoft.com/library/4bd5697d-bdde-4485-a05b-b3fea1403e16) object that is used to initialize a custom effect.</summary>
-        <returns>A handle to an [IMILBitmapEffect](http://msdn.microsoft.com/library/4bd5697d-bdde-4485-a05b-b3fea1403e16) object.</returns>
```

Comment:
This MSDN link re-directs to https://docs.microsoft.com/en-gb/previous-versions/windows/desktop/api/mileffects/nn-mileffects-imilbitmapeffect, which is a dead link. Not sure if that should be fixed?

Reply:
Thanks, @ChrisMaddock. I've reported the bad redirect.